cli.js

```diff
@@ -96,5 +97,18 @@ function handleFkillError(processes) {
 if (cli.input.length === 0) {
 	init(cli.flags);
 } else {
-	fkill(cli.input, cli.flags).catch(() => handleFkillError(cli.input));
+	let pids = [];
+	const promises = [];
+	for (let i = 0; i < cli.input.length; i++) {
+		if (cli.input[i][0] === ':') {
+			promises.push(portToProc(parseInt(cli.input[i].slice(1), 0)));
+		} else {
+			pids.push(cli.input[i]);
+		}
+	}
+	Promise.all(promises).then(promisedPids => {
+		promisedPids = promisedPids.filter(pid => pid);
```

Comment:
I'm not sure; I think if you ask to kill a valid target and an invalid target then the operation shouldn't fail; the valid target should be killed. But silently failing doesn't feel great, I agree. As a command line tool, rather than a library, perhaps it should be a little more permissive to user input? What do you think?